src/v/model/metadata.h

```diff
@@ -106,6 +106,28 @@ struct broker_endpoint final
     bool operator==(const broker_endpoint&) const = default;
     friend std::ostream& operator<<(std::ostream&, const broker_endpoint&);
 
+    static std::optional<ss::sstring> validate(const broker_endpoint& ep) {
+        if (ep.address.host() == "0.0.0.0") {
```

Comment:
>`validate` could possibly not be [static]

Any special reason? Feels intuitively non-member-y to me since we're just looking at a struct and saying 👍 / 👎

Reply:
eh i'm fine either way